scripts/jenkins/ardana/ansible/roles/setup_root_partition/tasks/main.yml

```diff
@@ -20,28 +20,9 @@
 # the file system layout settings defined in the input model.
 ---
 
-- name: Check for required tools
-  shell: >
-    command -v {{ item }}
-  register: _which_result
-  changed_when: False
-  failed_when:
-    - _which_result.rc > 1
-  loop: "{{ resize_tools }}"
-
-- name: Fail if required tools not found
-  fail:
-    # NOTE: blank line at start of msg text is intentional
-    msg: |
-
-      Required tools are missing from the SLES VM image:
-      {% for r in _which_result.results %}
-      {%   if r.rc %}
-          {{ r.item }}
-      {%   endif %}
-      {% endfor %}
-  when:
-    - _which_result.results | selectattr("rc") | list | length > 0
+- name: Install required tools
+  zypper:
```

Comment:
I suspect that will not always work, as zypper may not be available, because https://github.com/SUSE-Cloud/automation/blob/3cbcef225e2b3e103a0c98cd5abde4d05bf7d5c3/scripts/jenkins/ardana/ansible/bootstrap-vcloud-nodes.yml suggests that this may run on e.g. CentOS.
Do you have reason to think otherwise or have a different solution?

Reply:
Also, unless we have addressed the issue that only the deployer node has package repos configured to provide SLES or RHEL(CentOS) packages, there won't be any repos available from which to install packages at the point where this task list runs...

Basically the purpose of this code is really to sanity check that your backing image for CI deployments is setup with the appropriate packages.

Reply:
@JanZerebecki I can rework this to be more distro agnostic.

@rtamalin That suggests that we are placing the onus on the image build process itself to produce properly equipped images. In my opinion the source of truth so to speak would be better placed here. If something breaks, we now have to fix something in OBS/IBS.